### PR TITLE
Edit docs for compression-codec exchange property

### PR DESCRIPTION
**Pull request by Joelg96**
## Description

* Edit `compression-codec` exchange property default -> `LZ4`.
* Edit compression-codec mention and add a link on FTE docs page. 

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/admin/fault-tolerant-execution.md [hunk 1]
```diff
@@ -512,10 +512,10 @@ the property may be configured for:
   - HDFS
 :::
 
-It is recommended to set the `exchange.compression-codec` property to `LZ4` in
-the cluster's `config.properties` file, to reduce the exchange manager's overall
-I/O load. [](file-compression) is automatically performed and some details can
-be configured.
+To reduce the exchange manager's overall I/O load, the
+[](prop-exchange-compression-codec) configuration property is automatically
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.md:516]: a property can not be enabled .. it just is.. also does that link work?

File: docs/src/main/sphinx/admin/properties-exchange.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ network utilization.
 
 - **Type:** {ref}`prop-type-string`
 - **Allowed values:** `NONE`, `LZ4`, `ZSTD`
-- **Default value:** `NONE`
+- **Default value:** `LZ4`
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/properties-exchange.md:42]: Also delete the "Defaults to..." below since it is redundant.

Comment (mosabua) [reply docs/src/main/sphinx/admin/properties-exchange.md:42]: And confirm in the code

File: docs/src/main/sphinx/admin/fault-tolerant-execution.md (hunk 1 above)
Comment (Joelg96) [reply docs/src/main/sphinx/admin/fault-tolerant-execution.md:516]: Okay, and yes the link works

File: docs/src/main/sphinx/admin/properties-exchange.md (hunk 2 above)
Comment (Joelg96) [reply docs/src/main/sphinx/admin/properties-exchange.md:42]: From what I have found, the default is LZ4.

**Review by mosabua — APPROVED** [review]
Thanks. Looks good now.

**Pull request merged**